### PR TITLE
Arrow stick in wall

### PR DESCRIPTION
**Pull request by mattkmaier**
Created an arrow variant with a script that allows arrows to be stuck to game objects once there is a collision. 

Further enhancements would be objects that will bounce the arrow or let them pass. 

Arrows do not stick to other arrows to prevent stacking of arrows away from the wall.

### CODE REVIEW

File: Assets/Scripts/ArrowScript.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    {
+        /*
+         * Not sure what would be the most efficient could be done in one of multible ways
+         * arrowRB.sleep(); but if arrow is hit by another arrow they both fall
+         *
+         */
+        if (!collision.gameObject.CompareTag("arrow"))//To keep players from stacking arrows oddly
+        {
+            arrowRB.isKinematic = true;
+            arrowRB.velocity = Vector3.zero;
```
Comment (Kainkun) [inline Assets/Scripts/ArrowScript.cs:24]: I don't think this is necessary since kinematic rigidsbodys already don't move on their own

**Review by Kainkun — CHANGES_REQUESTED** [review]
Please change the name of your arrow variant to something that makes more sense like "sticky arrow" the name is up to you but If anyone else makes a prefab variant called "Arrow Variant" (which is likely because its the default name) there will definitely be a merge conflict.

File: Assets/Scripts/ArrowScript.cs (hunk 1 above)
Comment (mattkmaier) [reply Assets/Scripts/ArrowScript.cs]: Yep, now I know. I knew they weren't affected by external forces I didn't know they stopped  moving completely.

**Review by Kainkun — APPROVED** [review]
looks good, the script name might need to change later. I realized its also very general. Its ok for now though

**Pull request merged**